spec/array_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'array'
+
+describe Array do
+
+	it { is_expected.to respond_to(:new_inject) }
+
+	it 'expect new_inject to add numbers in array' do
+		expect([1,2,3,4].new_inject { |result, element| result + element }).to eq(10)
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Tab detected.